vignettes/how-to-update-released-site.Rmd

```diff
@@ -0,0 +1,265 @@
+---
+title: "How to update a released site"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to update a released site}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Checklist
+
+If you're already up-to-speed, here's the checklist:
+
+* Make sure your dev site looks the way you want before embarking on an update
+  of your released site. Here are good instructions for updating an existing
+  site for pkgdown 2.0.0, bs 5, and "tidyverse/tidytemplate":  
+  <https://tidytemplate.tidyverse.org>
+* Make sure you are using automatic development mode. In `_pkgdown.yml`:
+
+  ``` yaml
+  development:
+    mode: auto
+  ```
+* Push/pull to sync your default branch.
+* Create and checkout a branch based on the tag of your most recent GitHub/CRAN
+  release.
+  
+  Example for readr, whose latest release is 2.1.1:  
+  `git checkout -b update-pkgdown-2-1-1 v2.1.1`
```

Comment:
Maybe we should have a usethis helper for this? Then it could automatically push/pull?

Reply:
I guess it depends on whether this ends up feeling like a one-time thing for us.